browse/static/js/toggle-labs.js

```diff
@@ -13,32 +13,21 @@ $(document).ready(function() {
 
   var scripts = {
     "paperwithcode": $('#paperwithcode-toggle').data('script-url'),
-      "bibex": {
-          "url": "/bibex/bibex.js?20200709",
-          "container": "#bib-main"
-      },
-      "core-recommender": {
+    "bibex": {
+      "url": "https://static.arxiv.org/js/bibex-dev-tabs/bibex.js?20200709",
```

Comment:
In transitioning to static.arxiv.org for deploying this we'll need to:
1. Add instructions to the bibex git repo for deploying to static.arxiv.org
2. Remove the bibex deploys from e-prints/httpd/htdocs